backend/app/Repositories/UserRepository.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories;
+
+use App\User;
+use App\Repositories\Contracts\UserRepository as IUserRepository;
+
+final class UserRepository implements IUserRepository
```

Comment:
name implementation according to your persistence usage, e.g. EloquentUserRepository